core/lib/libtomcrypt/src/ciphers/sub.mk

```diff
@@ -11,7 +11,6 @@ srcs-y += aes_armv8a_ce.c
 srcs-y += aes_modes_armv8a_ce_a32.S
 else
 srcs-$(CFG_CRYPTO_AES) += aes.c
-srcs-$(CFG_CRYPTO_AES) += aes_tab.c
```

Comment:
Should we also delete the file from Git? I have no strong opinion, but I'm leaning towards deleting it.